File: algorithm/src/baekjoon/java/boj/BOJ11720.java
package baekjoon.java.boj;

import java.util.Scanner;

public class BOJ11720 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int count = Integer.parseInt(scanner.nextLine());

        String sNum = scanner.nextLine();

        int sum = 0;
        for (int i = 0; i < sNum.length(); i++) {
            sum += (int) (sNum.charAt(i) - 48);
        }

        System.out.println(sum);
    }
}

File: algorithm/src/baekjoon/kotlin/boj/BOJ11720.kt
package baekjoon.kotlin.boj

import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.lang.System.`in`
import java.lang.System.out

fun main() {
    val br = BufferedReader(InputStreamReader(`in`))
    val bw = BufferedWriter(OutputStreamWriter(out))
    val n = br.readLine()!!.toInt()
    val s = br.readLine()!!.toString()

    var sum: Int = 0
    for (i in 0 until n) {
        sum += s[i].code - 48
    }

    bw.write("$sum")
    bw.flush()
}